### DISCUSSION TITLE
Collection Editor special character issue

### DISCUSSION THREAD

**Discussion by amitpriyadarshi** [Sunbird-Ed/Community#24] [Bugs] [answered]
@sharathprasad4tarento in TextBook/Course creation/edit page, when a special character like (? or ! ) is passed in name fields, it shows an alert that `Special character! is not supported`. But it allows saving the collection object.

Also, the respective special characters are coming in the name text field, but in the TOC Textbook name trims the special character. Sharing the screenshot for ref. I think this is wrong behavior and data consistency should be there.
<img width="1199" alt="Screenshot 2022-06-16 at 5 50 52 PM" src="https://user-images.githubusercontent.com/6771856/174068579-7cdee281-308a-4eb9-b4f8-a263e6a42d5a.png">

**Comment by sharathkashyap** [comment, marked as answer]
@amitpriyadarshi Agreed that there has to be a proper validation for this which is consistent across. I'm afraid this issue may have to get reported onto Sunbird Co-Kreat as this building block. Request you to raise the Bug Request onto Sunbird Co-Kreat.

**Reply by vinukumar-vs** [reply]
@amitpriyadarshi this is valid issue. It is happening while opening any collection the default root node will be selected.  For the selected node (here default node) it is not showing the special characters. 
Also the same issue for other child nodes as well. While selecting any child node, you can see that special characters is not showing. But before selection for the child/root you can see the special characters will be showing.

**Reply by luckynvdu** [reply]
@vinukumar-vs can you please share the bug id created for the above issue?

cc @rhwarrier

**Reply by vinukumar-vs** [reply]
@amitpriyadarshi Can you move this conversation to [subird-knowlg](https://github.com/Sunbird-Knowlg/Community/discussions) & create the bug.
